angular-portfolio/src/app/award-detail/award-detail.component.html

```diff
@@ -0,0 +1,16 @@
+<div *ngIf="award">
+    <div class="flex-container">
+        <div *ngFor="let awd of awards" classs="photo"
+        (mouseenter)="enter(awd)"
```

Comment:
attributes should be indented. I'd also recommend but the closing angle bracket on the same line as the last attribute, but that's more personal preference.